core/templates/dev/head/pages/exploration_editor/editor_tab/unresolved_answers_overview_directive.html

```diff
@@ -0,0 +1,27 @@
+<style>
+  li.answerContainer {
+    vertical-align: text-bottom;
+  }
+  div.refreshContainer {
+    font-size: 0.8em;
+    text-align: right;
+  }
+  div.refreshContainer a {
+    font-weight: bold;
+  }
+</style>
+
+<div ng-show="unresolvedAnswersData !== null">
+  <strong>Unresolved Student Answers</strong>
+  <ul>
+    <li ng-repeat="data in unresolvedAnswersData" class="answerContainer">
+      <strong>x<[data.frequency]></strong>: <span ng-bind-html="data.answer"></span>
```

Comment:
Instead of x<[...]> I suggest "(<[...]> times)". The former reads as "eks 2" and it's not entirely obvious what it means.

Reply:
Done.